commit be2138efe0b4d86b7c40bfc219dcc103480f8b24 (refs/changes/27/2027/2)
Author: Marsell Kukuljevic <marsell@joyent.com>
Date:   2017-06-01T23:58:20+00:00 (2 years, 4 months ago)
    
    PUBAPI-1399: update CHANGES and version.
    Reviewed by: Trent Mick <trent.mick@joyent.com>

diff --git a/CHANGES.md b/CHANGES.md
index 901da0e..1de6817 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -1,5 +1,9 @@
 # node-ufds changelog
 
+## 1.3.0
+
+- Add support for noCache (disable cache) to get/listDcLocalConfig()
+
 ## 1.2.0
 
 - Add support for a clean install with node v4 (CAPI-522).
diff --git a/package.json b/package.json
index 75f5cdb..85d9c94 100644
--- a/package.json
+++ b/package.json
@@ -2,7 +2,7 @@
   "name": "ufds",
   "author": "Joyent (joyent.com)",
   "description": "SmartDataCenter UFDS Client API",
-  "version": "1.2.0",
+  "version": "1.3.0",
   "homepage": "https://github.com/joyent/sdc",
   "repository": {
     "type": "git",
